src/ukify/test/test_ukify.py

```diff
@@ -891,5 +891,70 @@ def test_key_cert_generation(tmp_path):
     assert 'Certificate' in out
     assert re.search(r'Issuer: CN\s?=\s?SecureBoot signing key on host', out)
 
+@pytest.mark.skipif(not slow_tests, reason='slow')
+def test_join_pcrsig(capsys, kernel_initrd, tmp_path):
+    if kernel_initrd is None:
+        pytest.skip('linux+initrd not found')
+    try:
+        systemd_measure()
+    except ValueError:
+        pytest.skip('systemd-measure not found')
+
+    ourdir = pathlib.Path(__file__).parent
+    pub = unbase64(ourdir / 'example.tpm2-pcr-public.pem.base64')
+
+    output = f'{tmp_path}/basic.efi'
+    args = [
+        'build',
+        *kernel_initrd,
+        f'--output={output}',
+        f'--pcr-public-key={pub.name}',
+        '--json=short',
+        '--policy-digest',
+    ] + arg_tools
+    opts = ukify.parse_args(args)
+    try:
+        ukify.check_inputs(opts)
+    except OSError as e:
+        pytest.skip(str(e))
```

Comment:
This seems a bit blunt. ukify throws `OSError` in this case
```
    # The API is not great.
    read = cp.read(filename)
    if not read:
        raise OSError(f'Failed to read {filename}')
```
but a lot of other things derive from `OSError`. The best way would probably be, if ukify defined and threw its own error, but you could also get away with checking the message, but  maybe not checking this at all would be possible.

Reply:
it's copied verbatim from other tests above, so if it's possible to change it, it can be done on all of them at the same time later